OMP Hobby ZMO integration documentation

tizian07 opened the pull request:
I added a documentation how to integrate a Auterion Skynode or Pixhawk flight controller into the OMP Hobby Tiltrotor VTOL.

sfuhrer commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+### Antennas and RC receiver
+
+One LTE antenna can be installed at the bottom of the vehicle. For that you can feed the antenna wire through the opening for the ESC heat-sink. The second antenna can be installed on the inside of the vehicle on the left side of the battery compartment. The RC receiver can also be placed at the left side of the battery compartment.
```
Specify that the LTE antennas are from the Skynode LTE module, and that builds with a Pixhawk instead would require some other telemetry link.

sfuhrer commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+### Select airframe
+
+Open QGC and go under vehicle setup. Search for SYS_AUTO in the parameters tab. Set SYS_AUTOCONFIG to "Reset parameters to airframe defaults" and SYS_AUTOSTART to 13300. Reboot the vehicle.
+
```
As we won't have the specific airframe selectable we should link a file with the params here.

sfuhrer replied:
Let's stream line this: My proposal is the following workflow:
1) open QGC and "Load from file" --> load base param file that we provide, which already includes the SYS_AUTOSTART, actuator mapping, tuning etc.
2) change actuator mapping if the user has a different wiring
3) trimming
4) sensor calibration, RC calibration
5) bench test, flying etc

hamishwillee replied:
@sfuhrer This works for me. The only thing is that I'd still like to know WHY you can't set the airframe from QGC - but are setting SYS_AUTOSTART. See https://github.com/PX4/PX4-user_guide/pull/2772#discussion_r1346491154

I'll merge once you guys have made this tidy.

sfuhrer replied:
That has been adapted by Tizian now. It's just a slightly different workflow

sfuhrer commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+- Motor 2: Front right motor should spin CCW
+- Motor 3: Rear motor should spin CCW
+
+If the motor spins in the wrong directions 2 of the 3 motor wires need to be swapped. The direction can not be changed in software since the ESCs are not using dshot.
```
```suggestion
If the motor spins in the wrong directions two of the three motor wires need to be swapped. The direction can not be changed in software since the ESCs are not using dshot.

```

hamishwillee replied:
FWIW What @sfuhrer has done is correct English prose. I do not enforce this particular case in PX4 docs - we use "pragmatic English prose". 

Why? Because there is no ambiguity in the use of numbers here, and it reduces effort to translate. However I do accept fixes like this if particular reviewers want them.

sfuhrer commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+- Check CG (There are markings underneath the wing)
+- Check actuator orientations and neutral trim
+- Check tilt rotor reactions in stabilized
```
```suggestion
- Check tilt rotor reactions in Stabilized flight mode

```

sfuhrer commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+- Check tilt rotor reactions in stabilized
+    - Yaw the vehicle to the right (nose to the right) -> right motor should tilt down
+    - Yaw the vehicle to the left (nose to the left) -> left motor should tilt down
+- Check actuator reactions in stabilized (switch transition switch to enable fw mode)
```
```suggestion
- Check actuator reactions in Stabilized (switch transition switch to enable FW mode)

```

hamishwillee replied:
FYI, note that where possible, good to link to the mode you talk about - that's even better than using the proper name. I have modified the comment above and will merge it.

sfuhrer commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+- Check kill switch
+- Arm in stabilized and check if motors react in the correct orientation but do not take off
+- Takeoff in stabilized and make some basic maneuvers
+- If everything went without any issue, takeoff in position mode and do a transition at ca. 25m. If something goes wrong switch back to MC mode as fast as possible.
```
```suggestion
- If everything went without any issue, takeoff in Position mode and do a transition at ca. 50m. If something goes wrong switch back to MC mode as fast as possible.
```

sfuhrer commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+    - Pitch the vehicle up (nose up) -> both elevons should go down
+    - Yaw the vehicle to the right (nose to the right) -> both elevons should go to the right, 
+- Check kill switch
+- Arm in stabilized and check if motors react in the correct orientation but do not take off
```
```suggestion
- Arm in stabilized and check if motors respond to the commands, e.g. roll left increases throttle on the right motor

```

hamishwillee commented:
I will look at this tomorrow.

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+Depending on the final takeoff weight the hover time might be limited, since there is not a lot of air circulation inside the fuselage when the vehicle is hovering.
+
+## Where to buy
```
We use "First Letter Capitalization". I'll modify

```suggestion
## Where to Buy

```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+- [GetFPV](https://www.getfpv.com/omphobby-zmo-z3-vtol-fpv-1200mm-arf-plane-kit-no-fpv-system.html)
+- [FoxtechFPV](https://www.foxtechfpv.com/zmo-pro-fpv-vtol.html)
+
+## Flight controller options
```
Note, there are many many more options than this. SO let's be more clear and state that these are options that have been tried.

```suggestion
## Flight Controller

The following options have been tested:
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+## Flight controller options
+
+- [Auterion Skynode evaluation kit](https://auterion.com/product/skynode/)
+- [Pixhawk 6C with PM02 V3](https://holybro.com/collections/autopilot-flight-controllers/products/pixhawk-6c?variant=42420293370045)
+- [Pixhawk 6C mini with PM02 V3](https://holybro.com/collections/autopilot-flight-controllers/products/pixhawk-6c-mini?variant=42783211290813)
```
FYI I'm going to modify these in a post-process to link to internal docs for the various components. They break less often than those for external vendors.

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+- [GPS F9P (included in Skynode eval. kit)](https://holybro.com/products/h-rtk-f9p-gnss-series?variant=41466787102909)
+- [GPS M9N (cheaper alternative to F9P)](https://holybro.com/collections/standard-gps-module/products/m9n-gps)
+- [Airspeed sensor (included in Skynode eval. kit)](https://www.dualrc.com/parts/airspeed-sensor-sdp33)
```
```suggestion
- [Airspeed sensor (included in Skynode eval. kit)](https://www.dualrc.com/parts/airspeed-sensor-sdp33) — recommended for improved safety and performance
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+- [GPS M9N (cheaper alternative to F9P)](https://holybro.com/collections/standard-gps-module/products/m9n-gps)
+- [Airspeed sensor (included in Skynode eval. kit)](https://www.dualrc.com/parts/airspeed-sensor-sdp33)
+- [Airspeed sensor (cheaper alternative)](https://holybro.com/products/digital-air-speed-sensor?pr_prod_strat=use_description&pr_rec_id=236dfda00&pr_rec_pid=7150470561981&pr_ref_pid=7150472462525&pr_seq=uniform)
+- [Lidar Lightware lw20 (included in Skynode eval. kit)](https://lightwarelidar.com/products/lw20-c-100-m)
```
```suggestion
- [Lidar Lightware lw20 (included in Skynode eval. kit)](https://lightwarelidar.com/products/lw20-c-100-m) (Optional)
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+- Sandpaper
+- 3D-Printer
+
+## Hardware integration
```
```suggestion
## Hardware Integration

```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+![ESC 03](../../assets/airframes/vtol/omp_hobby_zmo_fpv/esc-03.jpg)
+
+### Wing connector
```
```suggestion
### Wing Connector
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+![Wing connector 02](../../assets/airframes/vtol/omp_hobby_zmo_fpv/wing-connector-02.jpg)
+
+### Pixhawk adapter boards and BEC
```
```suggestion
### Pixhawk Adapter Boards and BEC
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+### Sensors
+
+#### Pitot tube
```
```suggestion
#### Pitot Tube
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+![Lidar 01](../../assets/airframes/vtol/omp_hobby_zmo_fpv/lidar-01.jpg)
+
+#### GPS
```
Yes, given that this is what is used

```suggestion
#### GPS/Compass
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+![USB Camera 01](../../assets/airframes/vtol/omp_hobby_zmo_fpv/camera-01.jpg)
+
+### Flight controller
```
```suggestion
### Flight Controller
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+![Flight controller 02](../../assets/airframes/vtol/omp_hobby_zmo_fpv/flight-controller-02.jpg)
+
+### Antennas and RC receiver
```
```suggestion
### Antennas and RC Receiver

```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+![SYS_AUTO](../../assets/airframes/vtol/omp_hobby_zmo_fpv/sys_auto.jpg)
+
+### Sensor calibration
```
```suggestion
### Sensor Calibration

```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+![Tilt Servo adjustment](../../assets/airframes/vtol/omp_hobby_zmo_fpv/tilt-limits-01.jpg)
+
+#### Control surfaces
```
```suggestion
#### Control Surfaces
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+Calibrate you RC controller and setup the flight mode switch.
+
+### Actuator setup
```
```suggestion
### Actuator Setup
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+Make sure the props are removed! The motors can start very easily in the actuators tab if a value gets changed in the wrong location.
+:::
+
+#### Tilt servos
```
```suggestion
#### Tilt Servos
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+![Servo trim](../../assets/airframes/vtol/omp_hobby_zmo_fpv/servo-trim.jpg)
+
+#### Motor direction and orientation
```
```suggestion
#### Motor Direction and Orientation
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,274 @@
+
+If the motor spins in the wrong directions 2 of the 3 motor wires need to be swapped. The direction can not be changed in software since the ESCs are not using dshot.
+
+## First flight
```
```suggestion
## First Flight
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,322 @@
+
+### Select Airframe
+
+1. Open QGC and go under vehicle setup.
```
@sfuhrer Why are we doing all this? Why not jus select a generic vtol tiltrotor in the Airframe selection screen, then configure the geometry? (i.e. remove one motor)

https://docs.px4.io/main/en/airframes/airframe_reference.html#vtol_vtol_tiltrotor_generic_quadplane_vtol_tiltrotor

I do think it would be good to provide a parameter file that can be applied as a diff if people want

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,322 @@
+- Check kill switch
+- Arm in [Stabilized mode](../flight_modes_fw/stabilized.md) and check if motors respond to the commands, e.g. roll left increases throttle on the right motor
+- Takeoff in stabilized and make some basic maneuvers
+- If everything went without any issue, takeoff in [Position mode](../flight_modes_fw/position.md) and do a transition at around 50m. If something goes wrong switch back to MC mode as fast as possible.
```
What about tuning? SHould we autotune?

sfuhrer replied:
No we'll provide recommended PX4 params including the tuning.

hamishwillee replied:
Sounds good, but what's the workflow? Are we expecting them to load airframe, then params then tidy up the actuator output mapping for how they wired it, or is this just something we provide at the end?

NOrmally we'd say "do all this, including tuning" then "here is a param file we provided earlier". Why? Because I'd like to make sure our tuning information is followable for this.

sfuhrer replied:
You're right, it's a bit contradicting atm. I've proposed a workflow that should work in https://github.com/PX4/PX4-user_guide/pull/2772/files#r1346984309

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,322 @@
+First make sure to set the [correct orientation of the flight controller](../config/flight_controller_orientation.md).
+This should be the default (`ROTATION_NONE`).
+
+Then calibrate the main sensors:
```
FYI I modified this a bit to explain in more detail what you do - more importantly to point to the relevant docs. NOTE, I am assuming mounting is as ROTATION_NONE

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,322 @@
+
+![Flight controller 02](../../assets/airframes/vtol/omp_hobby_zmo_fpv/flight-controller-02.jpg)
+
+### Antennas and RC Receiver
```
Yes, this section needs info about the options here - LTE is only for Skynode right?

tizian07 replied:
```suggestion
### Antennas and RC Receiver

:::note
If a Skynode is installed the LTE can be used as telemetry and video link. If a Pixhawk is used a different telemetry link will be needed. A basic example would be a [SiK Telemetry Radio](https://holybro.com/products/sik-telemetry-radio-v3?_pos=2&_sid=e5e1c6ba3&_ss=r).
:::

```
I added this note that a seperate telemetry link is needed if a Pixhawk is used.

hamishwillee replied:
Thanks. I'll modify a little - where possible better to link to internal docs - such as for Sik radio.

tizian07 commented:
Thanks a lot @sfuhrer and @hamishwillee for the review and commits! I reworked the open points. I also added the zip file with the stl files for the 3D printed parts.
Regarding the config file: My suggestion is to select the generic vtol airframe (sys_autostart 1230113) and then load the param file that we will provide. I can try this week and verify that everything works as expected.

sfuhrer commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -241,12 +247,14 @@ LTE antenna 2 and RC receiver
 1. Open QGC and go under vehicle setup.
 1. Search for [SYS_AUTO](../advanced_config/parameter_reference.md#SYS_AUTOSTART) in the parameters tab.
-1. Set [SYS_AUTOCONFIG](../advanced_config/parameter_reference.md#SYS_AUTOCONFIG) to "Reset parameters to airframe defaults" and [SYS_AUTOSTART](../advanced_config/parameter_reference.md#SYS_AUTOSTART) to `13300`.
+1. Set [SYS_AUTOCONFIG](../advanced_config/parameter_reference.md#SYS_AUTOCONFIG) to "Reset parameters to airframe defaults" and [SYS_AUTOSTART](../advanced_config/parameter_reference.md#SYS_AUTOSTART) to `1230113`.
```
13100 would be the generic tiltrortor VTOL, no?

hamishwillee replied:
My point in https://github.com/PX4/PX4-user_guide/pull/2772/files#r1345065617 is that you shouldn't do any of this. YOu just select the generic airframe by name in the config screen https://docs.px4.io/main/en/config/airframe.html then adjust the geometry.

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,330 @@
+### Antennas and RC Receiver
+
+:::note
+If a Skynode is installed the LTE can be used as telemetry and video link. If a Pixhawk is used a different telemetry link will be needed. A basic example would be a [SiK Telemetry Radio](https://holybro.com/products/sik-telemetry-radio-v3?_pos=2&_sid=e5e1c6ba3&_ss=r).
```
```suggestion
If a Skynode is installed the LTE can be used as telemetry and video link.
If a Pixhawk is used a different [telemetry link](../telemetry/README.md) will be needed.
An inexpensive example would be a [SiK Telemetry Radio](../telemetry/sik_radio.md).
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,332 @@
+- [Compass](../config/compass.md)
+- [Gyroscope](../config/gyroscope.md)
+- [Accelerometer](../config/accelerometer.md)
+- [Airspeed](../config/airspeed.html)
```
```suggestion
- [Airspeed](../config/airspeed.md)

```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,352 @@
+- [Servo cable extension cable male 30cm 10 pcs](https://www.getfpv.com/male-to-male-servo-extension-cable-twisted-22awg-jr-style-5-pcs.html)
+- [USB-C extension cable](https://www.digitec.ch/en/s1/product/powerguard-usb-c-usb-c-025-m-usb-cables-22529949?dbq=1&gclid=Cj0KCQjw2cWgBhDYARIsALggUhrh-z-7DSU0wKfLBVa8filkXLQaxUpi7pC0ffQyRzLng8Ph01h2R1gaAp0mEALw_wcB&gclsrc=aw.ds)
+- [3M VHB tape](https://www.amazon.in/3M-VHB-Tape-4910-Length/dp/B00GTABM3Y)
+- [3D-Printed mounts](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/vtol/omp_hobby_zmo_fpv/OMP-Hobby-ZMO-3D-Prints.zip)
```
FYI only -note this changed to a raw link rather than linking to the asset locally as you had done. Vuepress can't handle a direct link for some reason.

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,352 @@
+
+### ESCs
+
+1. Unsolder the ESC PWM-signal and ground pins and solder some servo extension wire to the pins.
```
FYI @tizian07 All the way through you will note I have modified this to a list. I prefer this because otherwise the instructions can be a "wall of text". By putting in list you can check that the instructions make sense in order, and put the images associated with each step where it is relevant.

It isn't "always better" but it can make these things easier to follow.

hamishwillee commented:
This is coming together. I'm happy with it except for the airframe/params part, as discussed here https://github.com/PX4/PX4-user_guide/pull/2772#discussion_r1346491154

sfuhrer commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -269,13 +269,10 @@ An inexpensive example would be a [SiK Telemetry Radio](../telemetry/sik_radio.m
-1) Set [SYS_AUTOCONFIG](../advanced_config/parameter_reference.md#SYS_AUTOCONFIG) to "Reset parameters to airframe defaults" and [SYS_AUTOSTART](../advanced_config/parameter_reference.md#SYS_AUTOSTART) to `1230113`.
-1) Reboot the vehicle.
-1) Load the [parameter-file]() up to the vehicle. To do so, go into the parameter tab and press on tools in the top-right corner. Select "Load from file.." and select the file.
+1) Select [Generic Quadplane VTOL](https://docs.px4.io/main/en/airframes/airframe_reference.html#vtol_standard_vtol_generic_standard_vtol) under the [Airframe](https://docs.px4.io/main/en/config/airframe.html) tab and click "Apply and Restart"
```
Should be "Generic Tiltrotor VTOL"

hamishwillee replied:
I think so.
```suggestion
1. Select [Generic Tiltrotor VTOL](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_generic_tiltrotor_vtol) under the [Airframe](https://docs.px4.io/main/en/config/airframe.html) tab and click **Apply and Restart**.
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,349 @@
+
+1) Open QGC and go under vehicle setup.
+1) Select [Generic Quadplane VTOL](https://docs.px4.io/main/en/airframes/airframe_reference.html#vtol_standard_vtol_generic_standard_vtol) under the [Airframe](https://docs.px4.io/main/en/config/airframe.html) tab and click "Apply and Restart"
+1) Select the [Parameters](https://docs.px4.io/main/en/advanced_config/parameters.html#finding-updating-parameters) tab and click on "Tools" in the top right corner. Select "Load from file..". Download the [parameter file](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/vtol/omp_hobby_zmo_fpv/OMP-Hobby-ZMO.params) and upload it to the flight controller.
+1) reboot the vehicle
```
Markdown expects lists to look like this `1.`  (though you can order them, you don't have to)
```suggestion
1. Select the [Parameters](https://docs.px4.io/main/en/advanced_config/parameters.html#finding-updating-parameters) tab and click on "Tools" in the top right corner. Select "Load from file..". Download the [parameter file](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/vtol/omp_hobby_zmo_fpv/OMP-Hobby-ZMO.params) and upload it to the flight controller.
1. Reboot the vehicle.
```

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,368 @@
+
+[Calibrate your RC Controller](../config/radio.md) and setup the [flight mode switches](../config/flight_mode.md).
+
+We recommend you assign RC switches for the set of modes defined in [Flight Mode Configuration > What Flight Modes and Switches Should I Set?](../config/flight_mode.md#what-flight-modes-and-switches-should-i-set).
+In particular you should assign a _VTOL Transition Switch_, _Kill Switch_, and a switch to select [Stabilized mode](../flight_modes_fw/stabilized.md) and [Position mode](../flight_modes_fw/position.md).
```
FYI added this because you later require certain switches to be set.

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,368 @@
+1. Select the [Airframe](https://docs.px4.io/main/en/config/airframe.html) tab
+1. Select [Generic Tiltrotor VTOL](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_generic_tiltrotor_vtol) from the _VTOL Tiltrotor_ group, and then and click **Apply and Restart**.
+
+### Load Parameters File
```
FYI, added to own section - allows explanation and linking.

Note strictly I guess you don't need the airframe loading if you have this, but I think it is good to show both.

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,368 @@
+
+Motors, control surfaces, and other actuators are configured in the QGroundControl [Actuator Configuration & Testing](../config/actuators.md).
+
+The [parameter file](#load-parameters-file) loaded previously means that this screen should already be correctly setup: you just need to adjust the trims for your particular vehicle.
+If motors/servos were connected to different outputs than suggested, you will need to change the output to function mappings in the actuator output section.
```
FYI this explains (again) why we're not doing geometry etc, and points out you might need to change wiring.

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,368 @@
+
+## First Flight
+
+- Check CG (There are markings underneath the wing)
```
Center of Gravity? Exactly what is it that you expect me to check?

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,368 @@
+  - Roll the vehicle to the right. The right aileron should go down.
+  - Pitch the vehicle up (nose up). Both elevons should go down.
+  - Yaw the vehicle to the right (nose to the right). Both elevons should go to the right.
+- Check kill switch
```
Why? How? A beginner might flight to 300m then press this. Do you think we could provide some more advice?

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,368 @@
+- Arm in [Stabilized mode](../flight_modes_fw/stabilized.md) and check if motors respond to the commands, e.g. roll left increases throttle on the right motor
+- Takeoff in [Stabilized mode](../flight_modes_fw/stabilized.md) and make some basic maneuvers
+- If everything went without any issue, takeoff in [Position mode](../flight_modes_fw/position.md) and do a transition at around 50m.
+  If something goes wrong switch back to multicopter mode as fast as possible.
```
```suggestion
  If something goes wrong switch back to multicopter mode as fast as possible (using the transition switch).

```

hamishwillee commented:
I've done one last pass and I'm pretty happy with this. A couple of questions in the First flight section.

hamishwillee commented on en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md:
```diff
@@ -0,0 +1,369 @@
+### Select Airframe
+
+1. Open QGC, select the **Q** icon, and then select **Vehicle Setup**.
+1. Select the [Airframe](https://docs.px4.io/main/en/config/airframe.html) tab
```
```suggestion
1. Select the [Airframe](../config/airframe.md) tab

```

hamishwillee commented:
Thanks very much. Looks great.
